Lib/asyncio/timeouts.py

```diff
@@ -23,14 +22,13 @@ class _State(enum.Enum):
     EXITED = "finished"
 
 
-@final
 class Timeout:
```

Comment:
Technically, runtime type checkers can use `__final__` attribute during their own analysis. But, I guess it is not a big deal. We never promised that it is indeed `final`, didn't we?

Reply:
I don't think runtime typecheckers can rely on typings in the standard library reliably at all, and that's still something they should be using the typeshed for. They would have other issues with the absence of `__final__` within the standard library where it already is only in the typeshed and not at runtime.